docs/root/source/timestamps.md

```diff
@@ -1,95 +0,0 @@
-# Timestamps in BigchainDB
```

Comment:
True.

It was a bit rushed to drop the entire file, not to mention it's nice documentation, but it will need a fairly comprehensive rewrite, as a large portion of it was related to the user provided timestamps.

I can rewrite it, but since it seems like the timestamps help file was born out of the ambiguity of the user provided transaction timestamp, what should be the new goals of the file?